src/components/accept-modal/AcceptModal.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { FC } from "react";
+import { getModalIds } from "../../core/utils/helpers/general";
+import Modal, { useModalButtonHandler } from "../../core/utils/modal";
+import { useText } from "../../core/utils/text";
+import { Button } from "../Buttons/Button";
+
+export interface AcceptModalProps {
+  accept: () => void;
+}
+
+const AcceptModal: FC<AcceptModalProps> = ({ accept }) => {
```

Comment:
Accept modal or /components/accept-modal is a quite generic namespace. Do you think the component is generic enough to be named like that or is it more tied to fees?

Reply:
can change the name to accept-fee-modal?